shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -334,6 +338,18 @@ - (void)updateWindowMetrics {
   FlutterEngineSendWindowMetricsEvent(_engine, &event);
 }
 
+- (void)scheduleOnRasterTread:(dispatch_block_t)block {
+  void* copy = Block_copy((__bridge void*)block);
```

Comment:
Is this to avoid ARC releasing block?

Reply:
Blocks are by default stack-allocated. Copy is necessary to bring it to heap.

Reply:
Gotcha